tests/storage/outofprocess_test.py

```diff
@@ -49,6 +51,23 @@ def oop_cleanup():
     oop.stop()
 
 
+@pytest.fixture(
+    scope="function",
+    params=[
+        userstorage.PATHS["mount-512"],
+        userstorage.PATHS["mount-4k"],
+    ],
+    ids=str,
+)
+def user_mount(request):
+    storage = request.param
+    if not storage.exists():
+        pytest.xfail("{} storage not available".format(storage.name))
+    file = Path(storage.path) / "file"
+    yield file
```

Comment:
So why it returns a filename?

If you want to return a file, call it user_file, same way it is used in other fixtures. But I don't think we should return a file here, since creating a file name
from the directory is trivial, and getting a directory is more flexiable. If we add a new tests that need to create 2 files, this fixture will be too specific and we will have to make it more generic and return the directory. So lets avoid this trouble by returning a directory like other user_mount fixture do.

Reply:
OK, I see your point to make it more extendable.

I returned a filename mostly because it was what all tests were doing, and I tried to avoid code repetition, having to do that every time in every test. But right, too specific behavior.